get-started/index.md

```diff
@@ -69,10 +69,10 @@ Use the search bar to find the **Red Hat build of Cryostat** catalog item.
   summary="Select the Cryostat Operator and click the Install button"
   image-name="`cryostat-operatorhub-install.png`"
 %}
-Choose your Operator installation mode:
+Choose your **Operator** installation mode:
 1. In **All Namespaces** installation mode, the **Cryostat Operator** instance will watch for **Cryostat** or
-**ClusterCryostat** Custom Resources (`CR`s) created in any Namespace and create corresponding Cryostat instances.
-2. In the **A specific namespace** installation mode, you must also select an installation Namespace, and the **Cryostat Operator** instance will only watch for **Cryostat** or **ClusterCryostat** instances created in that same Namespace.
+**ClusterCryostat** Custom Resources (**CR**s) created in any `Namespace` and create corresponding **Cryostat** instances.
+2. In the **A specific namespace** installation mode, you must also select an installation `Namespace`, and the **Cryostat Operator** instance will only watch for **Cryostat** or **ClusterCryostat** instances created in that same `Namespace`.
```

Comment:
Now we have sort of inverted the situation compared to before. It was inconsistent because of `CR` vs *Namespace*. Now it's **CR** and `Namespace`. It should be one or the other for both together.